### PR TITLE
[ADD][14.0] companyweb base

### PR DESCRIPTION
**Pull request by xavier-bouquiaux**
TODO
- [x] in the image URLs, change `acsone/l10n-belgium/14.0-companyweb_base/` to `OCAl10n-belgium/14.0/` before merging

### CODE REVIEW

**Review by rousseldenis — APPROVED** [review]
Code review. LGTM

File: companyweb_base/readme/INSTALL.rst [hunk 1]
```diff
@@ -0,0 +1,2 @@
+This module can be used in the usual way. It depends on the OCA
```
Comment (sbidoul) [inline companyweb_base/readme/INSTALL.rst]:
```suggestion
This module can be installed in the usual way. It depends on the OCA
```

File: companyweb_base/readme/USAGE.rst [hunk 2]
```diff
@@ -0,0 +1,45 @@
+This module comes with 2 security groups.
+
+.. figure:: https://raw.githubusercontent.com/acsone/l10n-belgium/14.0/companyweb_base/static/description/doc_security_groups.png
```
Comment (sbidoul) [inline companyweb_base/readme/USAGE.rst]:
```suggestion
.. figure:: https://raw.githubusercontent.com/OCA/l10n-belgium/14.0/companyweb_base/static/description/doc_security_groups.png
```

**Comment by sbidoul** [comment]
A couple of final tweaks then it's good (code review and functional test)

**Review by sbidoul — APPROVED** [review]

**Comment by sbidoul** [comment]
rebased and squashed

**Pull request merged**